src/test/java/com/wikia/webdriver/testcases/mercurytests/CategoryTest.java

```diff
@@ -21,14 +21,24 @@
   private void init() {
     this.category = new Category(driver);
 
-    new Navigate(driver).toPage(MercurySubpages.MAIN_PAGE);
+    new Navigate(driver).toPage(MercurySubpages.ARTICLE_WITH_CATEGORY_COMPONENT);
   }
 
-  @Test(groups = "mercury_category_expandAndNavigateToCategoryPage")
-  public void mercury_category_expandAndNavigateToCategoryPage() {
+  @Test(groups = "mercury_category_expandAndNavigateToCategoryPageWithArticle")
+  public void mercury_category_expandAndNavigateToCategoryPageWithArticle() {
     init();
 
-    category.toggle();
-    category.openCategoryPage(0);
+    category
+        .toggle()
+        .navigateToCategoryPageWithArticleAndWithMembers();
+  }
+
+  @Test(groups = "mercury_category_expandAndNavigateToCategoryPageWithoutArticle")
+  public void mercury_category_expandAndNavigateToCategoryPageWithoutArticle() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)